cypress/support/pages/product.pageObject.js

```diff
@@ -0,0 +1,19 @@
+import PageObject from '../PageObject';
+
+class ProductPageObject extends PageObject {
+  get addToCartButton() {
+    return cy.contains('Add to cart');
+  }
+
+  clickAddToCart() {
+    this.addToCartButton.click();
+  }
+
+  assertAlert(expectedMessage) {
+    cy.on('window:alert', (text) => {
+      expect(text).to.contains(expectedMessage);
```

Comment:
There is a minor issue here. The `expect` function should use `to.contain` instead of `to.contains`. The correct method is `to.contain` for checking if a string contains a substring.